docs/contributors/release.md

```diff
@@ -253,6 +253,135 @@ Now, the branch is ready to be used to publish the npm packages.
 
 Now, the npm packages should be ready and a patch can be created and committed into the corresponding WordPress SVN branch.
 
+
+### Standalone Package Releases 
+
+The following workflow is needed when packages require bug fixes or security releases to be published to _npm_ outside of a WordPress release cycle.
+
+Note: Both the `master` and `wp/trunk` branches are restricted and can only be _pushed_ to by the Gutenberg Core team.
+
+Identify the commit hashes from the pull requests that need to be ported from the repo `master` branch to `wp/trunk`
+
+The `wp/trunk` branch now needs to be prepared to release and publish the packages to _npm_.
+
+Open a terminal and perform the following steps:
+1. `git checkout master`
+1. `git pull`
+1. `git checkout wp/trunk`
+1. `git pull`
+
+Before porting commits check that the `wp/trunk` branch does not have any outstanding packages waiting to be published:
+1. `git checkout wp/trunk`
+1. `npm run publish:check`
+
+Now _cherry-pick_ the commits from `master` to `wp/trunk`, use `-m 1 commithash` if the commit was a pull request merge commit:
+1. `git cherry-pick -m 1 cb150a2`
+1. `git push` 
+
+Whilst waiting for the Travis CI build for  `wp/trunk` [branch to pass](https://travis-ci.com/WordPress/gutenberg/branches) identify and begin updating the `CHANGELOG.md` files:
+1. `git checkout wp/trunk`
+1. `npm run publish:check`
+> Example
+> ```shell
+> npm run publish:check
+> @wordpress/e2e-tests
+> @wordpress/jest-preset-default
+> @wordpress/scripts
+> lerna success found 3 packages ready to publish
+> ```
+
+Check the versions listed in the current `CHANGELOG.md` file, looking through the commit history of a package e.g [@wordpress/scripts](https://github.com/WordPress/gutenberg/commits/master/packages/scripts) and look out for _"chore(release): publish"_ and _"Update changelogs"_ commits to determine recent version bumps, then looking at the commits since the most recent release should aid with discovering what changes have occurred since the last release.
+
+Note: You may discover the current version of each package is not up to date, if so updating the previous released versions would be appreciated.
+
+Begin updating the _changelogs_ based on the [Maintaining Changelogs](https://github.com/WordPress/gutenberg/blob/master/packages/README.md#maintaining-changelogs) documentation and commit the changes:
+
+1. `git checkout wp/trunk`
+1. Update each of the `CHANGELOG.md` files
+1. Make a note of each packages new version, you will require these when it comes time to publish the packages with Lerna to npm:
+> Example
+> ```
+> @packages/e2e-tests/CHANGELOG.md           - @1.9.1
+> @packages/jest-preset-default/CHANGELOG.md - @5.3.1
+> @packages/scripts/CHANGELOG.md             - @6.1.1
+> ```
+5. Stage the _changelog_ changes `git add packages/`
+1. `git commit -m "Update changelogs"`
+1. Make a note of the commit hash of this commit
```

Comment:
Did you intend to number the list like this? The Markdown processor seems to tolerate it just as well and I could see how it makes it simpler to maintain, but there's an odd exception of the `5.` here, and it's also less readable in the raw markdown form.

Reply:
I did actually do this intentionally, I've been writing a lot of markdown lists recently, but it should be 4 nonetheless 😬

Also, as the w.org handbooks doesn't support this type of list format I will update those to 4,5,6.

Updated in https://github.com/WordPress/gutenberg/pull/19381/commits/a722aed28401a5bcd56a22af7816fd921cdc979f